axis_backend/Makefile

```
@@ -0,0 +1,54 @@
+.PHONY: help security-scan security-full test lint format
+
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (6).

(maxbodylength)